docker/developers/install-s3.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+set -e
+
+# Only allowed inside the docker container
+if [ -z ${WITHIN_DOCKER} ]; then
+  echo "ERROR: Cannot run within your normal environment."
+  echo "This command is only allowed to run inside the docker container connected to MinIO."
+  exit 1
+fi
+
+# Prevent aws cli from using interactive paging or failing when less isn't found
+# See: https://stackoverflow.com/questions/57953187/aws-cli-has-no-output
+export AWS_PAGER=
+
+export AWS_ENDPOINT_URL=${AWS_S3_ENDPOINT_URL}
+
+# Determine all the buckets that exist in the s3 path and create those buckets
+cd s3
+for path in `ls ./*/ -d`; do
+  path=$(basename "${path}")
+  bucket_name=${path//_/-}
+
+  if [ ! $(aws s3api head-bucket --bucket ${bucket_name}) ]; then
```

Comment:
...this may be incredibly obvious, but these are creating fake buckets via Minio, right? Is there any way to make that more clear to future readers?

Reply:
Sure. This is the normal `aws` command line usage to create buckets. It has nothing to do with MinIO outside of the fact that MinIO accepts the AWS CLI commands. If we want, we can of course link to the aws cli documentation: https://awscli.amazonaws.com/v2/documentation/api/latest/reference/index.html and specifically https://awscli.amazonaws.com/v2/documentation/api/latest/reference/s3api/index.html